editors/emacs/Portfile

```diff
@@ -44,9 +44,7 @@ if {$subport eq $name || $subport eq "emacs-app"} {
                     sha256  760382d5e8cdc5d0d079e8f754bce1136fbe1473be24bb885669b0e38fc56aa3 \
                     size    65007481
 
-    # our dbus is autolaunched by launchd, so disable the check that it's running
-    patchfiles      patch-src_dbusbind.c.diff \
-                    patch-configure.diff
+    patchfiles      patch-configure.diff
```

Comment:
If removal of `patch-src_dbusbind.c.diff` is intentional, please remove the file too.

Reply:
I'll put it back.